apps/web/src/components/Announcement/constants.ts

```diff
@@ -0,0 +1,29 @@
+// key: based on UserExperienceType schema
+
+export const ANNOUNCEMENT_CONTENT = [
+  {
+    key: 'MultiGalleryAnnouncement',
+    title: 'Multi Gallery',
+    description: 'You can now create multiple galleries and add them to your profile.',
+    date: '2023-03-14T04:56:26.154845Z',
+    link: '/mint/mementos',
+  },
+  {
+    key: 'EmailUpsell',
+    title: 'Email Upsell',
+    description: 'You can now add your email address to your profile. ',
+    date: '2023-03-26T04:56:26.154845Z',
+  },
+  {
+    key: 'MaintenanceFeb2023',
+    title: 'Maintenance Feb 2023',
+    description: 'We will be performing maintenance on the site on Feb 2023.',
+    date: '2023-03-14T04:56:26.154845Z',
+  },
+  {
+    key: 'TwitterConnectionOnboardingUpsell',
+    title: 'Connect Twitter',
+    description: 'Connect your Twitter account to your profile.',
+    date: '2023-03-14T04:56:26.154845Z',
+  },
+];
```

Comment:
This is just dummy content. Lmk what kind of content should we put first

Reply:
let's make a new experience flag for mementos, and remove the rest. will need to add the experience to the backend- here's the [loom video](https://www.loom.com/share/97a3402a018941a491410278fdb789f7) for doing that

```js
export const ANNOUNCEMENT_CONTENT = [
  {
    key: 'UpsellMintMemento4',
    title: 'Now Minting: Blooming Connections',
    description: 'Gallery Memento #4 is now available for minting, a beautiful and symbolic representation of the growing network within the Gallery community.',
    date: '2023-03-27T13:00:00.154845Z',
    link: '/mint/mementos',
  },
];
```